src/main/external-resources/renderers/FetchTV.conf

```diff
@@ -0,0 +1,40 @@
+#----------------------------------------------------------------------------
+# Profile for Fetch TV (Mighty and Mini) @ https://www.fetchtv.com.au/
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+
+RendererName = Fetch TV
+RendererIcon = Fetch-TV.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# User-Agent: Takin/3.0.0 (Linux arm ; U; en), FetchTV_STB_BCM7252S/3.7.7244 (FetchTV, M616T, Wireless)
+# USER-AGENT: Linux/3.14.28-1.6, HTTP/1.0, Portable SDK for HTTP devices
+# ============================================================================
+#
+
+UserAgentSearch = FetchTV
+
+TranscodeVideo = MPEGTS-H264-AC3
+TranscodeAudio = MP3
+TranscodeFastStart = true
+MediaInfo = true
+
+# Supported video formats (see https://www.fetchtv.com.au/pdf/Fetch_TV_Media_Hub_Quick_Start_Guide_-_Media_Servers.pdf)
+Supported = f:avi       v:xvid                                       a:mp3                                         m:video/avi
```

Comment:
Better to delete `(see https://www.fetchtv.com.au/pdf/Fetch_TV_Media_Hub_Quick_Start_Guide_-_Media_Servers.pdf)`